packages/cactus-plugin-ledger-connector-fabric-socketio/package.json

```diff
@@ -11,12 +11,12 @@
     "prepare-docker-build": "cp -af ../../yarn.lock ./dist/yarn.lock"
   },
   "dependencies": {
-    "@types/node": "14.17.32",
-    "body-parser": "1.17.2",
-    "config": "3.3.7",
-    "cookie-parser": "1.4.6",
-    "debug": "4.1.1",
-    "express": "4.15.5",
+    "@types/node": "^14.14.5",
```

Comment:
@micoferdinand98 Why go back to enabling auto-upgrade?

Reply:
sorry peter it was changed when I was resolving the merge conflicts.

Reply:
@micoferdinand98 No worries. It still has auto-upgrade on it. The version of `@types/node` has also changed relative to `main`, I'm thinking that's also a conflict resolution mistake, but please confirm separately.